xptifw/unit_test/CMakeLists.txt

```diff
@@ -1,10 +1,11 @@
-if (NOT EXISTS ${XPTI_SOURCE_DIR})
-  message (FATAL_ERROR "Undefined XPTI_SOURCE_DIR variable: Must be set for tests to work!")
+if (NOT EXISTS ${XPTI_DIR})
```

Comment:
Where is this set when doing build as part of llvm?

Reply:
@aelovikov-intel XPTI_SOURCE_DIR and LLVM_EXTERNALXPTIFW_SOURCE_DIR are set in buildbot/configure.py. The main xptifw/CMakeFiles.txt in uses the above variables to set XPTI_DIR based on whether it is being build as a part of LLVM or outside of it.